core/domain/wipeout_domain_test.py

```diff
@@ -26,13 +26,13 @@
 class PendingDeletionRequestUnitTests(test_utils.GenericTestBase):
     """Tests for topic domain objects."""
 
-    def setUp(self):
+    def setUp(self) -> None:
         super(PendingDeletionRequestUnitTests, self).setUp()
         self.signup('a@example.com', 'A')
         self.signup('b@example.com', 'B')
-        self.user_id_a = self.get_user_id_from_email('a@example.com')
+        self.user_id_a = self.get_user_id_from_email('a@example.com') # type: ignore[no-untyped-call]
```

Comment:
Is this a temporary warning suppression or something that needs to be around for a while? If the latter, why?

Reply:
This is temporary, when `get_user_id_from_email` is typed (when we will be handling the tests folder), MyPy will alert us.

Reply:
Ah excellent--thanks for the context!